### PR TITLE
add movenet lite deploy support

### PR DESCRIPTION
**Pull request by zhiboniu**
add movenet lite deploy support

### CODE REVIEW

File: deploy/movenet_lite/README.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+|--valid_targets|指定模型可执行的backend，默认为arm。目前可支持x86、arm、opencl、npu、xpu，可以同时指定多个backend(以空格分隔)，Model Optimize Tool将会自动选择最佳方式。如果需要支持华为NPU（Kirin 810/990 Soc搭载的达芬奇架构NPU），应当设置为npu, arm|
+
+更详细的`paddle_lite_opt`工具使用说明请参考[使用opt转化模型文档](https://paddle-lite.readthedocs.io/zh/latest/user_guides/opt/opt_bin.html)
+
+`--model_file`表示inference模型的model文件地址，`--param_file`表示inference模型的param文件地址；`optimize_out`用于指定输出文件的名称（不需要添加`.nb`的后缀）。直接在命令行中运行`paddle_lite_opt`，也可以查看所有参数及其说明。
+
+
+#### 2.1.3 转换示例
+
+下面以PaddleDetection中的 `PP-YOLO-tiny` 模型为例，介绍使用`paddle_lite_opt`完成预训练模型到inference模型，再到Paddle-Lite优化模型的转换。
```
Comment (jerrywgz) [inline deploy/movenet_lite/README.md]: 这里需要把PP-YOLO-tiny改下

File: deploy/movenet_lite/src/preprocess_op.cc [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  for (int i = 0; i < rc; ++i) {
+    cv::extractChannel(*im, cv::Mat(rh, rw, CV_32FC1, base + i * rh * rw), i);
+  }
+}
+
+void Resize::Run(cv::Mat* im, ImageBlob* data) {
+  auto resize_scale = GenerateScale(*im);
+  cv::resize(
+      *im, *im, cv::Size(), resize_scale.first, resize_scale.second, interp_);
+  // int ypad = static_cast<int>(im->rows % 32);
```
Comment (jerrywgz) [inline deploy/movenet_lite/src/preprocess_op.cc]: remove comment

File: deploy/movenet_lite/README.md (hunk 1 above)
Comment (zhiboniu) [reply deploy/movenet_lite/README.md]: done

File: deploy/movenet_lite/src/preprocess_op.cc (hunk 2 above)
Comment (zhiboniu) [reply deploy/movenet_lite/src/preprocess_op.cc]: done

**Review by jerrywgz — DISMISSED** [review]

**Pull request closed without merging**